frontend/packages/client/src/components/ProposalCreate/StepTwo/StepTwo.js

```diff
@@ -123,15 +123,15 @@ const StepTwo = ({
               <Input
```

Comment:
on line 83 we need to set choices to remove more than 2 options if user has created more than 2 text options and clicks on image: 
`````
 let choices = choicesTemp;
    if (tabOption === 'visual') {
      choices = choicesTemp.slice(0, 2);
      setValue('choices', choices);
    } else {
````

Reply:
@germanurrus For this I think we are okay - ranked choice voting is not supported for Image Voting, so the user currently can only do two choices.